src/styles/HeroComponent.scss

```diff
@@ -111,3 +111,7 @@ $heroBackgroundColor: #f7f7f7;
 .react-syntax-highlighter-line-number {
   user-select: none;
 }
+
+.option-descr {
```

Comment:
What does this end up applying to? (Also missing newline after, if this should stay)

Reply:
This shouldn't be in this PR, I'll remove it.